Source/for_helpers/for_determining_artifact_destination/for_when_cwd_is_in_the_correct_area/given/a_path_with_no_feature.js

```diff
@@ -0,0 +1,12 @@
+import { all_dependencies } from '../../given/all_dependencies';
```

Comment:
Path kinda breaks the `given`, `when`, `then` flow of BDD with the `for_when*` pattern. 
`For` is supposed to give the thing that it is testing. It would be ok to just drop the `for_` in the path and start with `when_*` as that is the scenario. Reads out a bit weird when it says `for_`

I see you've done this around - could you clean up this throughout?